internal/monitor/exporter.go

```diff
@@ -106,3 +109,59 @@ func CloseOpenTelemetryCollectorExporter() {
 	}
 	ocExporter = nil
 }
+
+var prometheusExporter *prometheus.Exporter
+var prometheusServer *http.Server
+
+// EnablePrometheusCollectorExporter starts exporting monitoring metrics for
+// the Prometheus to scrape on the given port.
+func EnablePrometheusCollectorExporter(port int) error {
```

Comment:
Calling this method multiple times with different ports will lead to memory leaks. We should probably throw an error if someone calls this method when there are preexisting server and exporter running.

Reply:
Done.